docs/throughput-anomaly-detection.md

```diff
@@ -110,13 +138,24 @@ in table format, run:
 
 ```bash
 $ theia throughput-anomaly-detection retrieve tad-1234abcd-1234-abcd-12ab-12345678abcd
-id                                      sourceIP        sourceTransportPort     destinationIP   destinationTransportPort        flowStartSeconds        flowEndSeconds          throughput                     algoCalc            anomaly
-1234abcd-1234-abcd-12ab-12345678abcd    10.10.1.25      58076                   10.10.1.33      5201                            2022-08-11T06:26:54Z    2022-08-11 08:24:54     10004969097.000000000000000000  4.0063773860532994E9        true
-1234abcd-1234-abcd-12ab-12345678abcd    10.10.1.25      58076                   10.10.1.33      5201                            2022-08-11T06:26:54Z    2022-08-11 08:06:54     4005703059.000000000000000000   1.0001208294655691E10       true
-1234abcd-1234-abcd-12ab-12345678abcd    10.10.1.25      58076                   10.10.1.33      5201                            2022-08-11T06:26:54Z    2022-08-11 08:34:54     50007861276.000000000000000000  3.9735065921281104E9        true
+id                                      sourceIP        sourceTransportPort     destinationIP   destinationTransportPort        flowStartSeconds        flowEndSeconds          throughput              aggType         algoType        algoCalc                anomaly
+1234abcd-1234-abcd-12ab-12345678abcd    10.10.1.25      58076                   10.10.1.33      5201                            2022-08-11T06:26:54Z    2022-08-11T08:06:54Z    4.005703059e+09         e2e             ARIMA           1.0001208441920074e+10  true
```

Comment:
I might be wrong, but is `e2e` an unexpected aggType? I expect it should be one of external, pod and svc.

Reply:
I agree, it should be None as aggType